docs/development-tips.md

```diff
@@ -34,6 +23,11 @@ After installing these projects as sibling directories to the `cfgov-refresh` re
     Clone and install the projects as siblings to `cfgov-refresh`,
     so that they share the same parent directory (`~/Projects` or similar).
 
+In addition to the aforementioned list,
```

Comment:
Maybe move this up near the line about `optional-public`? Encountering this at the end is a bit unclear as these apps aren't included in that requirements file.